File: Lecture005 Bitwise Operators, For Loops, Operator Precedence & Variable Scoping/forFib.java
class forFib {
    public static void main(String[] args) {
        int n = 10;

        int a = 0;
        int b = 1;

        System.out.print(a + " " + b + " ");

        for (int i = 1; i <= n; i++) {
            int nextNumber = a + b;
            System.out.print(nextNumber + " ");

            a = b;
            b = nextNumber;
        }
    }
}

File: Lecture005 Bitwise Operators, For Loops, Operator Precedence & Variable Scoping/forFib.cpp
#include<iostream>
using namespace std;

int main() {

    int n = 10;

    int a = 0;
    int b = 1;
    cout<<a <<" " <<b<<" ";
    for(int i = 1; i<=n; i++ ) {
        
        int nextNumber = a+b;
        cout<<nextNumber<<" ";

        a = b;
        b = nextNumber;
    }
    return 0;
}